v3/src/components/graph/components/graph-component.tsx

```diff
@@ -17,24 +17,33 @@ import {isGraphContentModel} from "../models/graph-content-model"
 import {Graph} from "./graph"
 import {DotsElt} from '../../data-display/d3-types'
 import {AttributeDragOverlay} from "../../drag-drop/attribute-drag-overlay"
+import {PixiPoints} from "../utilities/pixi-points"
 import "../register-adornment-types"
 
-export const GraphComponent = observer(function GraphComponent({tile}: ITileBaseProps) {
+export const GraphComponent = observer(function GraphComponent({ tile }: ITileBaseProps) {
   const graphModel = isGraphContentModel(tile?.content) ? tile?.content : undefined
 
   const instanceId = useNextInstanceId("graph")
   const { data } = useDataSet(graphModel?.dataset)
   const layout = useInitGraphLayout(graphModel)
   // Removed debouncing, but we can bring it back if we find we need it
   const graphRef = useRef<HTMLDivElement | null>(null)
-  const {width, height} = useResizeDetector<HTMLDivElement>({ targetRef: graphRef })
+  const { width, height } = useResizeDetector<HTMLDivElement>({ targetRef: graphRef })
   const dotsRef = useRef<DotsElt>(null)
+  const pixiPointsRef = useRef<PixiPoints>()
+  // TODO PIXI: PJ: probably should be fixed and become a ref, as memoization is meant for performance optimization
+  // and it's not guaranteed to be maintained across renders.
```

Comment:
Can also use something like https://www.npmjs.com/package/use-memo-one.

Reply:
It looks nice. I'm wondering if there's any advantage/difference to using refs?